src/tests/lib/components/Input.spec.ts

```diff
@@ -412,7 +412,7 @@ describe("Input", () => {
       }
 
       fireEvent.input(input, { target: { value: 0.00000001 } });
```

Comment:
Aren't input events always with strings?
Does clicking the "Max" button generate an input event? I would think it sets the value directly?

Reply:
True, good point! Replaced with `$set`.